manual/src/cmds/intf-c.etex

```diff
@@ -1210,7 +1210,7 @@ value alloc_list_int(int i1, int i2)
 
   r = caml_alloc(2, 0);                   /* Allocate a cons cell */
   Store_field(r, 0, Val_int(i2));         /* car = the integer i2 */
-  Store_field(r, 1, Val_int(0));          /* cdr = the empty list [] */
+  Store_field(r, 1, Val_emptylist);       /* cdr = the empty list [] */
```

Comment:
Unfortunately, `Val_emptylist` is not documented in the manual. So if we wanted to do this change, we need to document the macro first.

Reply:
Do you have a correct place to document this? This seemed like the place where this would make sense to even be introduced.

Reply:
How about at L613:

https://github.com/ocaml/ocaml/blob/d57938d2ffbe2c60a892cc3355e81524308954cb/manual/src/cmds/intf-c.etex#L613

Reply:
I think this is where the other aliases are introduced, so I added it there.